modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-catalog-impl/rest-openapi.yaml

```diff
@@ -1237,6 +1249,1067 @@ paths:
+                      description:
+                          Channel external reference code
+                      type: string
+                  style: simple
+                - in: query
+                  name: accountId
+                  required: false
+                  schema:
+                      description:
+                          Account Id
+                      example: 30130
+                      format: int64
+                      minimum: 0
+                      type: integer
+            requestBody:
+                content:
+                    application/json:
+                        schema:
+                            $ref: "#/components/schemas/WishList"
+                    application/xml:
+                        schema:
+                            $ref: "#/components/schemas/WishList"
+                required: true
+            responses:
+                201:
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/WishList"
+                        application/xml:
+                            schema:
+                                $ref: "#/components/schemas/WishList"
+                    description:
+                        Created
+                400:
+                    description:
+                        Invalid input
+                401:
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/Error"
+                        application/xml:
+                            schema:
+                                $ref: "#/components/schemas/Error"
+                    description:
+                        Authentication information is missing or invalid
+                404:
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/Error"
+                        application/xml:
+                            schema:
+                                $ref: "#/components/schemas/Error"
+                    description:
+                        The specified resource was not found
+                500:
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/Error"
+                        application/xml:
+                            schema:
+                                $ref: "#/components/schemas/Error"
+                    description:
+                        Unexpected error
+            summary: Creates a WishList.
+            tags: ["WishList"]
+    "/channels/{channelExternalReferenceCode}/products/{productExternalReferenceCode}/product-options/by-externalReferenceCode/{externalReferenceCode}/product-option-values":
```

Comment:
same, put the by-externalReferenceCode everywhere and rename the externalReferenceCode to productOptionExternalReferenceCode param